test/CMakeLists.txt

```
@@ -48,6 +48,7 @@ set(TEST_TARGETS
   test_utils_tests
   utils_tests
   device_selection_tests
+  divergence_check_tests
```

```suggestion
  divergence_checker_tests
```

(Same for test utils)